sql-plugin/src/main/scala/com/nvidia/spark/rapids/higherOrderFunctions.scala

```diff
@@ -336,9 +339,9 @@ case class GpuArrayExists(
   }
 
   private def imputeFalseForEmptyArrays(
-    transformedCV: cudf.ColumnView,
-    result: cudf.ColumnView
-  ): GpuColumnVector = {
+                                         transformedCV: cudf.ColumnView,
+                                         result: cudf.ColumnView
+                                       ): GpuColumnVector = {
```

Comment:
Not sure why this and `transformListColumnView` indentation of arguments was changed here?